build/fpga-regmap/src/lib.rs

```diff
@@ -155,19 +163,22 @@ fn write_reg_fields(children: &[Node], prefix: &str, output: &mut String) {
 {prefix}        pub const {inst_name}: u8 = 0b{mask:08b};",
             )
             .unwrap();
+
             // Deal with optional encoded Enums on this field
             match encode {
                 Some(x) => {
+                    let encode_name = inst_name.clone() + "_Encoded";
                     writeln!(
                         output,
                         "
-{prefix}        use num_derive::{{ToPrimitive, FromPrimitive}};
-{prefix}        #[derive(Copy, Clone, Eq, PartialEq, FromPrimitive, ToPrimitive)]
+{prefix}        #[derive(Copy, Clone, Eq, PartialEq)]
 {prefix}        #[allow(dead_code)]
 {prefix}        #[allow(non_camel_case_types, clippy::upper_case_acronyms)]
```

Comment:
Also, re: naming, I kind of wonder if maybe we should just downcase the register name and emit `RegisternameEncoded`, instead of `REGISTERNAME_Encoded`? Up to you, though.

Reply:
I think this is a good suggestion to make this whole crate feel much more Rust-y rather than the SHOUTING AT US.